pkg/antctl/raw/multicluster/init.go

```diff
@@ -112,6 +115,23 @@ func initRunE(cmd *cobra.Command, args []string) error {
 	// Declare ClusterSet init succeeded, even if there is a failure later when creating the
 	// member token or writing the join config file.
 	fmt.Fprintf(cmd.OutOrStdout(), "Successfully initialized ClusterSet %s\n", initOpts.clusterSet)
+	fmt.Fprintf(cmd.OutOrStdout(), "You can run command \"antctl mc get joinconfig -n %s\" to print the parameters needed for a member cluster to join the ClusterSet.\n", initOpts.namespace)
+
+	var tokenSecret *corev1.Secret
+	if initOpts.createToken {
+		if err := common.CreateMemberToken(cmd, initOpts.k8sClient, defaultToken, initOpts.namespace, &createdRes); err != nil {
+			fmt.Fprintf(cmd.ErrOrStderr(), "Failed to create member token\n")
```

Comment:
Maybe we can add a message here to notify user that they can retry with `antctl mc create membertoken` later?

Reply:
Sounds like a good idea. Added.